pom.xml

```diff
@@ -145,9 +145,7 @@
       <rabbitmq.http-client.version>3.5.0.RELEASE</rabbitmq.http-client.version>
       <!-- Newer version contains new rule that we violate. Needs to be fixed first -->
       <hibernate-validator.version>6.0.20.Final</hibernate-validator.version>
-      <!-- Newer community version does not support MySQL 5.6 anymore -->
-      <flyway.version>5.2.4</flyway.version>
```

Comment:
and we don't need to track this anymore? or would we end up with the same situation with future versions?

Reply:
Yes, since we moved to MySQL 5.7 pinning flyway version is not anymore required and can rely on spring specific version.
**would we end up with the same situation with future versions** -incase Flyway decides to remove support for 5.7 in their future versions then YES... we might have to do something similar like this until we move to their supported version :)